### ISSUE TITLE
4.0.x is not compatible with Chef 13

### ISSUE THREAD

**Issue by trivoallan** [sous-chefs/rundeck#170]
### Cookbook version
4.0.5

### Chef-client version
13.2.20

### Platform Details
Debian 8.8

### Scenario:
Using test-kitchen with `require_chef_omnibus: 13`, when trying to converge the rundeck::server i get the following error :

```
Chef::Mixin::Template::TemplateError (undefined method `rundeck' for #<Chef::Node::Attribute:0x00000002a48920>) on line #42:
       
        40: #
        41: #
        42: <% if node.rundeck['custom_framework_config'] %>
        43: <% node.rundeck['custom_framework_config'].sort.each do |key, value| %>
        44: <%= key %>=<%= value %>
```

The error those not happen with `require_chef_omnibus: 12`

**Comment by ev0ldave** [comment]
This issue is resolved in commit 3badf8a and is being released with 4.1.0.... getting that out ASAP

**Comment by gsreynolds** [comment]
The cookbook is now Chef 13 compatible and being tested using latest 13.x, so I'll close this issue.
